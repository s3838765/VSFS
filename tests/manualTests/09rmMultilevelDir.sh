#! /usr/bin/sh
cp 09rmMultilevelDir.notes 09rmMultilevelDirTemp.notes
../../VSFS rmdir 09rmMultilevelDirTemp.notes dir1/dir2/
cp 09rmMultilevelDirTemp.notes 09rmMultilevelDir.out
rm 09rmMultilevelDirTemp.notes
