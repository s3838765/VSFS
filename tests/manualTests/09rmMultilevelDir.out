NOTES V1.0
=dir1/
@rootfile1
 Some data
#dir1/dir2/
@dir1/file1
#dir1/dir2/dir3/
#dir1/dir2/file3
@rootfile2
 Some more file data
